### PR TITLE
[starts #162952197] Added login page

### PR DESCRIPTION
**Pull request by a-braham**
#### What does this PR do?
 * The pool request creates login form that a user fill to login.

#### Description of Task to be completed?
  **Have the following features complete**
  * Login form should be simple and descriptive
  * Page should be responsive
  * Should have all the required fields; username and password

#### How should this be manually tested?
  * Clone the repository
  * Open the the signup.html file with chrome
  * Test for responsiveness
 
#### Screenshots
screenshot from 2019-01-04 11-44-32

#### What are the relevant pivotal tracker stories?
  #162952122

### CODE REVIEW

**Comment by kalyakiplangat** [comment]
Looks good

File: ui/login.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                <p class="log-footer" style="margin-left:20%">Don't have an account?<a class="meetup-url" style="text-decoration:none; color: #2196F3; font-size:120%" href="#"> Sign up</a></p>
+            </form>
+    </div>
+    
+    <footer class="footer">
+        <p>&copy; 2019 Questioner Andela</p>
+    </footer>
+</body>
+
+</html>
```
Comment (Ogollah) [inline ui/login.html]: You should add a new line

**Comment by Ogollah** [comment]
Consider adding a new line.
Good work on the ui.

File: ui/login.html (hunk 1 above)
Comment (a-braham) [reply ui/login.html]: To push the footer up?

**Comment by colMike** [comment]
Good work

**Comment by a-braham** [comment]
> Good work

Thank you

**Pull request merged**